tests/acceptance/bootstrap/AuthAppContext.php

```diff
@@ -83,6 +84,7 @@ public function userHasCreatedAppTokenWithExpirationTime(string $user, string $e
 			["expiry" => $expiration]
 		);
 		$this->featureContext->theHTTPStatusCodeShouldBe(200, "", $response);
+		$this->createdAuthAppToken = $response;
```

Comment:
you can store only the token
```suggestion
		$this->createdAuthAppToken = <store-token-only>;
```